kvpress/presses/key_rerotation_press.py

```diff
@@ -0,0 +1,79 @@
+# SPDX-FileCopyrightText: Copyright (c) 1993-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import inspect
+from dataclasses import dataclass
+
+import torch
+from torch import nn
+from transformers.models.llama.modeling_llama import rotate_half
+
+from kvpress.presses.base_press import BasePress
+from kvpress.presses.scorer_press import ScorerPress
+
+
+@dataclass
+class KeyRerotationPress(BasePress):
+    """
+    Rerotate keys to have a uniform RoPE representation of keys after pruning.
+    This method is used in several key-value cache compression methods, such as
+    - SinkCache implementation in Hugging Face's transformers library
```

Comment:
The current implementation of `StreamingLLMPress` is not correct. Correct implementation would be `KeyRerotationPress(StreamingLLMPress(...))`. Could you update the docstring of StreamingLLMPress to tell it OR fix StreamingLLMPress ?

Reply:
I added a comment in the docstring of `StreamingLLMPress`